.prettierrc

```
@@ -0,0 +1,6 @@
+{
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": true,
+  "arrowParens": "avoid"
```

I'd keep the default always unless you really prefer without.
```suggestion
  "arrowParens": "always"
```